source/_components/light.knx.markdown

```diff
@@ -31,16 +31,36 @@ light:
     brightness_address: '1/0/11'
 ```
 
-Configuration variables:
-
-- **address** (*Required*): KNX group address for switching the light on and off.
-- **name** (*Optional*): A name for this device used within Home Assistant.
-- **brightness_address** (Optional): KNX group address for dimming light.
-- **state_address** (*Optional*): separate KNX group address for retrieving the switch state of the light.
-- **brightness_state_address** (*Optional*): separate KNX group address for retrieving the dimmed state of the light.
-- **color_address** (*Optional*): separate KNX group address for setting the color of the light.
-- **color_state_address** (*Optional*): separate KNX group address for retrieving the color of the light.
+{% configuration %}
+address:
+  description: KNX group address for switching the light on and off.
+  required: true
+  type: string
+name:
```

Comment:
Optional variables are shown in the config sample.

Reply:
Have them removed